src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card" >
+      <img class="card__img"
+      src="./images/imac.jpeg"
+      alt="imac"
+      >
+      <div class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+      <div class="card__product-code">Product code: 195434</div>
+
+      <div class="grade">
+        <div class="grade__stars4">
+          <div class="grade__stars4-star"></div>
+          <div class="grade__stars4-star"></div>
+          <div class="grade__stars4-star"></div>
+          <div class="grade__stars4-star"></div>
+          <div class="grade__stars4-star"></div>
+        </div>
```

Comment:
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```

Reply:
in BEM you can have block inside block and it's a case where we can use it
4 active stars it's modificator, so you can write it like {block}--{modificator}